website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,894 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0, codename `Cobalt`. This release includes
+a huge number of new features and improvements, which we were perfecting over
+the last couple of months.
```

Comment:
> This release includes a huge number of new features and improvements, ~which we were~ perfecting over the last couple of months.
------
This release includes a huge number of new features and improvements, which we've been perfecting over the last couple of months.

Reply:
How about formulating the introduction like this?

---

We are excited to announce the release of Metals v0.8.0, codename `Cobalt`. This release includes a large number of new features and improvements, which have been under development since September 2019.

* Commits since last release: 727
* Merged PRs: 207
* Contributors: N
* Closed issues: 82
* New features: M

For full details:  https://github.com/scalameta/metals/milestone/9?closed=1

Reply:
Fixed